cmake/DGtalConfigGenerator.cmake

```diff
@@ -7,29 +7,39 @@ export(TARGETS  DGtal FILE "${PROJECT_BINARY_DIR}/DGtalLibraryDepends.cmake")
 # export(PACKAGE DGtal)
 # Not working on cmake 2.6, I remove this option but keep the codeline (DC)
 
-# Create a DGtalConfig.cmake file for the use from the build tree
-set(DGTAL_INCLUDE_DIRS "${PROJECT_SOURCE_DIR}/src" "${PROJECT_BINARY_DIR}/src")
-set(DGTAL_LIB_DIR "${PROJECT_BINARY_DIR}/src")
-set(DGTAL_CMAKE_DIR "${PROJECT_BINARY_DIR}")
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
 
+# Provide HINTS to find_dependency in DGtalConfig.cmake if DGTAL_CONFIG_HINTS
+# is ON. Disable it when deploying.
```

Comment:
what is find_dependency ?

Reply:
Is like find_package, but with a better error message, if the package is missing. Only used in `FooConfig.cmake` files.

Reply:
👍